sys/shell/commands/shell_commands.c

```diff
@@ -209,6 +213,9 @@ const shell_command_t _shell_command_list[] = {
 #ifdef MODULE_USB_BOARD_RESET
     {"bootloader", "Reboot to bootloader", _bootloader_handler},
 #endif
+#ifdef MODULE_BENCHMARK_UDP
+    {"benchmark", "UDP benchmark", _benchmark_udp_handler},
```

Comment:
I am not sure if there will be any other benchmark in RIOT (e.g.: the for the developing tcp), maybe this should not capture this generic term but use its modulname (unique within RIOT) for the shell interface.
```suggestion
    {"benchmark_udp", "UDP benchmark", _benchmark_udp_handler},
```

Reply:
how about `bench_udp` so it's less to type